tensorboard/java/org/tensorflow/tensorboard/vulcanize/Vulcanize.java

```diff
@@ -405,6 +405,10 @@ private static Node replaceNode(Node oldNode, Node newNode) {
     return newNode;
   }
 
+  private static Node removeNode(Node node) {
+    return replaceNode(node, new TextNode("", node.baseUri()));
```

Comment:
I assume this will be changed after merging #2430?

Reply:
Yep.